text/0000-glimmer-components.md

```diff
@@ -0,0 +1,835 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Glimmer Components
+
+## Summary
+
+Glimmer components are a simpler, more ergonomic, and more declarative approach
+to building components. They represent the sum of multiple years of design and
+feature work by the community, which stemmed from the original RFCs and
+discussions surrounding "angle-bracket components".
+
+This RFC proposes adding Glimmer components to Ember's public API, and making
+them the default new app experience in Ember Octane.
+
+## First, a Bit of History
+
+As components became the standard for Single Page Apps (SPAs) several years
+ago, and the Ember community began adopting them in earnest and converting from
+Ember 1's primarily MVC oriented approach, there were many small and large
+issues that cropped up with Ember's component API and usage: `{{curly-bracket}}`
```

Comment:
This formatting of the list of issues is non-standard, as far as I can tell. Grammatically, I think you'd better off replacing the semicolons with commas. Additionally, the word after a semicolon shouldn't be capitalized unless it's a proper noun.

Reply:
Actually, on re-reading, I understand why you went for semicolons here, but the point about capitalizing the word after a semicolon still stands.